framework/include/utils/MathFVUtils.h

```diff
@@ -161,6 +161,8 @@ enum class InterpMethod
 {
   /// (elem+neighbor)/2
   Average,
+  /// 1/(gc/elem+(1-gc)/neighbor)
+  HarmonicAverage,
   /// (gc*elem+(1-gc)*neighbor)+gradient*(rf-rf')
   SkewCorrectedAverage,
```

Comment:
Unfortunately, now that I see we've added gc to the others, we should include gc in the `Average` as well. It is not simply 1/2 weights all the time

Reply:
Sounds good to me, will take care of that today.